res/adjectives.txt

```diff
@@ -41,8 +41,10 @@ non-standard
 cached
 stored
 formatted
-assembling
-disassembling
 distributed
 virtual
 augmented
+binary
+classical
+literary
+multiple
```

Comment:
Questionable compatibility; may not mesh well with several nouns. (e.g. "compiling multiple cache", "turbo-encabulating mutliple Java")